source/_integrations/lutron.markdown

```diff
@@ -22,6 +23,8 @@ Presently, there's only support for communicating with the [RadioRA 2](http://ww
 
 When configured, the `lutron` integration will automatically discover the rooms and their associated switches/dimmers as configured by the RadioRA 2 software from Lutron. Each room will be treated as a separate group.
 
+This component supports importing Areas defined by the Lutron RadioRA2 system into Home Assistant.  You can enable this feature in the integration UI.
+
 To use Lutron RadioRA 2 devices in your installation, add the following to your `configuration.yaml` file using the IP address of your RadioRA 2 main repeater:
```

Comment:
Isn't this deprecated now?

Reply:
I was looking through other component documentation trying to figure out what the standard is for describing the configuration migration to config_flow.  If you have a good example I'm happy to model it after that.  This came as a result of looking at some of the ones that I know still support configuration.yaml and have config_flow like rainmachine and tplink.